views/admin_list_rhapsody.handlebars

```diff
@@ -108,8 +108,8 @@
                                     </td>
                                     <td>{{rhapsody.date}}</td>
                                     <td> 
-                                        <a href="/admin/rhapsody/view/{{rhapsody.id}}" class="btn btn-info"> view</a>
-                                        <a href="/admin/rhapsody/edit/{{rhapsody.id}}" class="btn btn-info"> Edit</a> 
+                                        <a href="/admin/edit-rhapsody-realities/{{rhapsody.id}}" class="btn btn-info"> view</a>
```

Comment:
view link you lead you to a page that looks like the rhaspody page a user sees, we can skip this for now though

Reply:
coolio